contents/docs/session-replay/privacy.mdx

```diff
@@ -24,7 +24,8 @@ You can control the masking more granularly by using `maskInputFn` to customize
 ```ts
 posthog.init('<ph_project_api_key>', {
     session_recording: {
-        maskAllInputs: true, // Important - this needs to be true for the below function to be called
+        // `maskInputFn` only applies to selected elements, so make sure to set this to true if you want this to work globally
+        maskAllInputs: true,
```

Comment:
is it maskInputFn or maskAllInputs?

Reply:
The comment is meant to indicate that `maskInputFn` only runs for inputs that have been marked as needing masking. So `maskAllInputs: true` is required for this case.

Please feel free to make changes to what you think is clearer as the whole motivation of this PR is that some users didnt get it